packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -113,11 +113,24 @@ const defaultChatItemMessageContainer = (overlapAvatarAndMessage: boolean): Comp
 /**
  * @private
  */
-export const defaultChatItemMessageContainerNoOverlap = makeStyles(defaultChatItemMessageContainer(false));
-/**
- * @private
- */
-export const defaultChatItemMessageContainerOverlap = makeStyles(defaultChatItemMessageContainer(true));
+export const defaultChatItemMessageContainerStyles = makeStyles({
+  body: {
+    marginRight: '0rem',
+    zIndex: CHAT_MESSAGE_ZINDEX,
+    '& msft-mention': {
+      color: '#D83B01',
```

Comment:
We should use theme values for colors (tokens if possible or something like was done in `const errorTextColor = 'var(--errorText)';`)